tests/PySys/apt_plugin/apt_remove/run.py

```diff
@@ -39,17 +40,17 @@ def execute(self):
         )
 
     def validate(self):
-        self.assertGrep ("before.out", 'rolldice', contains=True)
-        self.assertGrep ("after.out", 'rolldice', contains=False)
+        self.assertGrep("before.out", "rolldice", contains=True)
+        self.assertGrep("after.out", "rolldice", contains=False)
```

Comment:
Does this kind of change really matter?

Reply:
No not really, I think this slipped in by using black as code formatter.